### PR TITLE
Introduce VoyageAI reranker module

### PR DESCRIPTION
**Pull request by fzowl**
### Review checklist

- [x] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [x] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: modules/reranker-voyageai/module.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		return errors.Wrap(err, "init cross encoder")
+	}
+
+	return nil
+}
+
+func (m *ReRankerVoyageAIModule) initAdditional(ctx context.Context, timeout time.Duration,
+	logger logrus.FieldLogger,
+) error {
+	apiKey := os.Getenv("VOYAGE_API_KEY")
```
Comment (antas-marcin) [inline modules/reranker-voyageai/module.go:66]: please use the same env variable as in vectorizer module which is: `VOYAGEAI_APIKEY`

**Comment by antas-marcin** [comment]
@fzowl in order for us to be able to merge your PR, you need to agree to our CLA. Simply writing a comment with "I agree with CLA" will enable us merging your PR

**Comment by fzowl** [comment]
I agree with CLA

**Comment by fzowl** [comment]
Reopen

**Review by antas-marcin — APPROVED** [review]

**Pull request merged**